src/node.cc

```diff
@@ -546,14 +562,17 @@ static void PlatformInit(ProcessInitializationFlags::Flags flags) {
 #else
     // Tell V8 to disable emitting WebAssembly
     // memory bounds checks. This means that we have
-    // to catch the SIGSEGV in TrapWebAssemblyOrContinue
+    // to catch the SIGSEGV/SIGBUS in TrapWebAssemblyOrContinue
     // and pass the signal context to V8.
     {
       struct sigaction sa;
       memset(&sa, 0, sizeof(sa));
       sa.sa_sigaction = TrapWebAssemblyOrContinue;
       sa.sa_flags = SA_SIGINFO;
       CHECK_EQ(sigaction(SIGSEGV, &sa, nullptr), 0);
+#ifdef  __APPLE__
```

Comment:
Maybe we could leave a `TODO` comment here about dropping the `__APPLE__` conditionals in the next major version? It's probably a good idea to align Node.js behavior across OSes in the long run.

Reply:
It depends on V8 because linux and osx raise difference signal and node should handle them

Reply:
Yes, but right now this code means that Node.js installs itself as a signal handler for `SIGBUS` on one OS but not on other POSIX systems, which means that on that OS `RegisterSignalHandler()` needs to be used to handle that signal but not on the others. That’s probably okay, but aligning behavior is usually a good thing – worst case it just adds an extra unnecessary step for handling `SIGBUS`, right?